autogen/versions.tf.tmpl

```diff
@@ -19,9 +19,9 @@ terraform {
 
   required_providers {
 {% if beta_cluster %}
-    google-beta = "~> 2.18"
+    google-beta = "~> 2.18.0"
```

Comment:
These changes should be reverted; 2.20.1 fixes the IAM problem we were experiencing.

Reply:
reverted